File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0;
        int count2=0;
        for(int i=0;i<s.size();i++){
            if(s[i]!=' '){
                count++;
            count2 =count;
            }
            else{
                count =0;
            }
        }
        return count2;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count=0;
        int j=s.length()-1;
        while(j>=0 && s.charAt(j)== ' ')j--;
        while(j>=0 && s.charAt(j)!= ' '){
            count++;
            j--;
            }
           
        
    return count;  
    }
}